releases/8.0.0.md

```diff
@@ -80,21 +83,36 @@ JavaScript:
     npm install
     ```
 
-2. Within your project, with your Python 3 virtual environment activated:
+1. Update `.gitignore`:
+    1. Remove `.tsconfig-paths.json` and `.frontend-configuration-settings.json`
+    2. Add `frontend_configuration`
+
+1. Replace the `generate_frontend_configuration` import statement in `apps.py`:
+    ```
+    from arches.settings_utils import generate_frontend_configuration
```

Comment:
I toyed with importing this into the old location so that projects wouldn't need to update, but things broke, so 👍